ParisBNF/abb/BNFabb158.xml

```diff
@@ -47,7 +47,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                ወበእንተዝ፡ ወፅአ፡ ሊቀ፡ ጳጳሳት፡ ወበጽሐ፡ ወርእየ፡ 
                                ዘንተ፡ ኵሎ፡ ተአምረ፡                                
                             </incipit>                     
-                     <note>Divided into 25 chapters; the text not used in the edition 
+                     <note>Divided into 25 chapters. The text was not used in the edition 
```

Comment:
we should distinguish text and manuscript. is this text not present in the edition or was this manuscript not used?